recipes/dggrid/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dggrid" %}
+{% set version = "7.7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sahrk/DGGRID/archive/v{{ version}}.tar.gz
+  sha256: 1dd74919798b393146cfc96cb6969d30660e4e260707426e6754b46b5de81a2a
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake >=3.12,<4
+    - make  # [linux]
+  host:
+    - libgdal
+    - shapelib
```

Comment:
It's good that you are depending on preexisting conda-forge packages, but you need to actually link to them. The logs show that you are not linking to shapelib dynamically.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=652966&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1972

Reply:
cmake is now configured to load and dynamically link shapelib from the host, the vendored version 1.55 src is kept for legacy compile without conda environment)